api/hasura/event_triggers/createVouchedUser.ts

```diff
@@ -0,0 +1,80 @@
+import assert from 'assert';
+
+import type { VercelRequest, VercelResponse } from '@vercel/node';
+
+import { gql } from '../../../api-lib/Gql';
+import { EventTriggerPayload } from '../../../api-lib/types';
+import { verifyHasuraRequestMiddleware } from '../../../api-lib/validate';
+
+async function handler(req: VercelRequest, res: VercelResponse) {
+  const {
+    event: { data },
+  }: EventTriggerPayload<'nominees', 'INSERT'> = req.body;
```

Comment:
@CryptoGraffe this is the event trigger that creates the user from the nomination, we could create a similar one that triggers when `vouches` `INSERT` with the same business logic